src/bcf/header.rs

```diff
@@ -180,6 +180,49 @@ impl HeaderView {
 
         Ok((_type, length))
     }
+
+    /// Convert string ID (e.g., for a `FILTER` value) to its numeric identifier.
+    pub fn id2int(&self, id: &[u8]) -> Result<u32, IdError> {
+        unsafe {
+            match htslib::bcf_hdr_id2int(
+                self.inner,
+                htslib::BCF_DT_ID as i32,
+                ffi::CString::new(id).unwrap().as_ptr() as *const i8
+            ) {
+                -1 => Err(IdError::UnknownID(str::from_utf8(id).unwrap().to_owned())),
+                i => Ok(i as u32),
+            }
+        }
+    }
+
+    /// Convert integer representing an identifier (e.g., a `FILTER` value) to its string
+    /// name.bam
+    pub fn int2id(&self, id: i32) -> Vec<u8> {
```

Comment:
Take above newtype here, call this function `id_to_name`.

Reply:
OK